dev/WindowsAppRuntime_DLL/WindowsAppRuntime_DLL.vcxproj

```diff
@@ -171,7 +172,7 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <EnableUAC>false</EnableUAC>
-      <AdditionalDependencies>onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>onecore.lib;onecoreuap.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <ModuleDefinitionFile>WindowsAppRuntime.def</ModuleDefinitionFile>
       <DelayLoadDLLs>Microsoft.Internal.FrameworkUdk.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
>DelayLoadDLLs [](http://example.com/codeflow?start=7&length=13)

shell32.lib presumably brings in shell32.dll. If so, I think that dll should be added to the DelayLoadDlls (for all flavors).

Reply:
Thank you for your guidance. The modifications related to shell32.lib and shell32.dll have been added to all configurations